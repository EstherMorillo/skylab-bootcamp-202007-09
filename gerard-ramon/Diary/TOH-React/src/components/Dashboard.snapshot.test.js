import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './Dashboard';

describe('Dashboard snapshot', () => {
	const tree = renderer.create(<Dashboard />);

	it('Should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
