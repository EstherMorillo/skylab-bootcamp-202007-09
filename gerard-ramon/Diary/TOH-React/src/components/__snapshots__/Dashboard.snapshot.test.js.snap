// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard snapshot Should match 1`] = `
<ul
  className="heroDashboard"
/>
`;
